logging/log_collection_forwarding/log-forwarding.adoc

```diff
@@ -2,42 +2,49 @@
 include::_attributes/common-attributes.adoc[]
 include::_attributes/attributes-openshift-dedicated.adoc[]
 [id="log-forwarding"]
-= About log forwarding
+= About log collection and forwarding
 :context: log-forwarding
 
 toc::[]
 
-By default, the {logging} sends container and infrastructure logs to the default internal log store defined in the `ClusterLogging` custom resource. However, it does not send audit logs to the internal store because it does not provide secure storage. If this default configuration meets your needs, you do not need to configure the Cluster Log Forwarder.
+Administrators can create `ClusterLogForwarder` resources that specify which logs are collected, how they are transformed, and where they are forwarded to.
 
-To send logs to other log aggregators, you use the {product-title} Cluster Log Forwarder. This API enables you to send container, infrastructure, and audit logs to specific endpoints within or outside your cluster. In addition, you can send different types of logs to various systems so that various individuals can access each type. You can also enable Transport Layer Security (TLS) support to send logs securely, as required by your organization.
+`ClusterLogForwarder` resources can be set up to forward container, infrastructure, and audit logs to specific endpoints within or outside of a cluster. Transport Layer Security (TLS) is supported so that log forwarders can be configured to send logs securely.
 
-[NOTE]
-====
-To send audit logs to the default internal Elasticsearch log store, use the Cluster Log Forwarder as described in xref:../../logging/config/cluster-logging-log-store.adoc#cluster-logging-elasticsearch-audit_cluster-logging-log-store[Forward audit logs to the log store].
-====
+Administrators can also authorize RBAC permissions that define which service accounts and users can access and forward which types of logs.
 
-When you forward logs externally, the {logging} creates or modifies a Fluentd config map to send logs using your desired protocols. You are responsible for configuring the protocol on the external log aggregator.
+include::modules/log-forwarding-modes.adoc[leveloffset=+1]
 
-// unused files - either include or delete
-// cluster-logging-log-forwarding-disable.adoc
+[id="log-forwarding-enabling-multi-clf-mode"]
+== Enabling multi log forwarder mode for a cluster
 
-include::modules/cluster-logging-collector-log-forwarding-about.adoc[leveloffset=+1]
+To use multi log forwarder mode, you must create a service account and cluster role bindings for that service account. You can then reference the service account in the `ClusterLogForwarder` resource to control access permissions.
```

Comment:
Note: The cluster-logging operator must be deployed as global operator.

Reply:
Is there something outside of the usual default installation settings for the Operator that needs to be done for this?
E.g. do users need to follow https://docs.openshift.com/container-platform/4.11/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-global-namespaces_olm-adding-operators-to-a-cluster or similar, or is it enough as long as their Operator is installed in **All namespaces** / the `openshift-operators` global namespace?

Reply:
I don't think this matters given we have to change the deployed manifest to watch all namespaces; that alone makes it a global operator.  AFAIK there is no way for someone deploying the operator to change the installMode.  We will continue to install into "openshift-logging" with no intention or expectation to install elsewhere